workflows/k8-wavefront-collector/workflow.yaml

```diff
@@ -111,6 +115,8 @@ spec:
               metadata:
                 name: k8-pod-delete
                 namespace: {{workflow.parameters.appCurrentNamespace}}
+                labels:
+                  context: kube-system/{{workflow.parameters.appLabel}}
```

Comment:
context: "{{workflow.parameters.appNamespace}}_{{workflow.parameters.appLabel}}"

Reply:
We can not use, since it is crossing chars limit and as per discussed  have updated with label value with namespace